src/web_server.cpp

```diff
@@ -709,7 +709,7 @@ handleScheduleGet(MongooseHttpServerRequest *request, MongooseHttpServerResponse
     response->setCode(200);
     serializeJson(doc, *response);
   } else {
-    response->setCode(404);
+    response->setCode(200);
```

Comment:
The `404` is correct in this case, the requested event does not exist and the `404` allows you to detect this